### PR TITLE
Shivani Madhan: Menu Displayed instead of Placeholder — Endpoint for meals

### LINKED ISSUE #13: Menu Displayed instead of Placeholder — Endpoint for meals
Create a service that will get all of the meals served in a particular dining commons on a particular date

Using the api endpoint described here: https://developer.ucsb.edu/apis/dining/dining-menu

Then build a controller endpoint /api/diningcommons/:date-time/:dining-commons-code the returns this information.

### PR DESCRIPTION
**Pull request by clevermonkey16**
Created three Swagger endpoints under DiningMenuAPIController

/api/dining/getDays: Returns a list of days with meal service based on data from the given Dining Menu API
/api/dining/getCommons: Returns a list of dining commons serving meals on a given date (takes in date as input)
/api/dining/getMeals: Returns a list of meals being served at a given dining commons on a given date (takes in dining commons and date as input)

[Dokku Link](https://dining-shivanimadhan.dokku-13.cs.ucsb.edu)

Closes #13

### CODE REVIEW

**Review by emiz39 — DISMISSED** [review]
getDays is not working on swagger

**Comment by shivanimadhan** [comment]
Swagger is working on localhost but not on dokku

File: src/main/java/edu/ucsb/cs156/dining/controllers/DiningMenuAPIController.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    return ResponseEntity.ok().body(body);
+  }
+
+  // @Operation(summary = "Load days with meal service into database from API")
+  // @PreAuthorize("hasRole('ROLE_ADMIN')")
+  // @PostMapping("/loadDays")
+  // public List<DiningMenuAPI> loadDays() throws Exception {
+  //   List<DiningMenuAPI> savedDays = diningMenuAPIService.loadAllDays();
+  //   return savedDays;
+  // }
```
Comment (pconrad) [inline src/main/java/edu/ucsb/cs156/dining/controllers/DiningMenuAPIController.java:58]: https://ucsb-cs156.github.io/topics/code_reviews/commented_out_code.html#why-should-commented-out-code-be-removed

**Review by pconrad — CHANGES_REQUESTED** [review]
https://ucsb-cs156.github.io/topics/code_reviews/commented_out_code.html#why-should-commented-out-code-be-removed

**Comment by Anteater10** [comment]
LGTM

**Review by pconrad — APPROVED** [review]
LGTM

**Pull request merged**